### PR TITLE
working basic/rbac/vault solution - some improvements still to come

### PR DESCRIPTION
**Pull request by mccullya**

### CODE REVIEW

File: examples/basic/confluent/control-centre.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+---
+apiVersion: platform.confluent.io/v1beta1
+kind: ControlCenter
+metadata:
+  name: controlcenter
+spec:
+  tls:
+    secretRef: tls-group1
```
Comment (sionsmith) [inline examples/basic/confluent/control-centre.yaml:8]: Do we need this in the basic deployment? Not sure?

**Review by sionsmith — CHANGES_REQUESTED** [review]
Can we remove the vault-pki-engine folder? Will cause merge hell with me :)

File: examples/basic/confluent/control-centre.yaml (hunk 1 above)
Comment (mccullya) [reply examples/basic/confluent/control-centre.yaml]: nah, you're right... in light of our conv the other day, I've changed these to 'auto generated'... there is still a reference to the secret-tls in kustomize, but should be able to remove all of that once AC-58 is sorted.

**Comment by mccullya** [comment]
I've reverted the commit with the PKI stuff, so that should be OK now....

**Review by sionsmith — APPROVED** [review]

**Pull request merged**